apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/CharSequenceToContainCreatorSamples.kt

```diff
@@ -0,0 +1,257 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.expect
+import kotlin.test.Test
+
+class CharSequenceToContainCreatorSamples {
+
+    @Test
+    fun value() {
+        expect("ABC") toContain o exactly 1 value "A"
+        expect("ABBBC") toContain o atLeast 2 value "B"
+
+        fails {
+            expect("AAAAAA") toContain o atMost 3 value "A"
+        }
+    }
+
+    @Test
+    fun valueIgnoringCase() {
+        expect("ABC") toContain o ignoring case value "a"
+        expect("AbbbC") toContain o ignoring case value "B"
+
+        fails {
+            expect("AAAAAA") toContain o ignoring case value "B"
+        }
+    }
+
+    @Test
+    fun valueIgnoringCaseWithChecker() {
+        expect("ABC") toContain o ignoring case exactly 1 value "A"
+        expect("AAABBC") toContain o ignoring case atMost 3 value "b"
+        expect("aBBBCD") toContain o ignoring case atLeast 1 value "A"
+
+        fails {
+            expect("AAAAAABBBB") toContain o ignoring case atMost 3 value "A"
+        }
+        fails {
+            expect("AAABBBb") toContain o ignoring case exactly 3 value "b"
+        }
+        fails {
+            expect("AAAAAABBBB") toContain o ignoring case atLeast 3 value "D"
+        }
+    }
+
+    @Test
+    fun values() {
+        expect("ABC") toContain o exactly 1  the values("A","B","C")
+        expect("AAABC") toContain o atMost 3 the values("A", "B", "C")
+        expect("ABBBCD") toContain o atLeast 1 the values("A", "B", "C", "D")
+
+        fails {
+            expect("AAAAAABBBB") toContain o atMost 3 the values("A", "B")
+        }
+        fails {
+            expect("AAABBBB") toContain o exactly 3 the values("A", "B")
+        }
+        fails {
+            expect("AAAAAABBBB") toContain o atLeast 3 the values("A", "B", "C")
+        }
+    }
+
+    @Test
+    fun valuesIgnoringCase() {
+        expect("AbC") toContain o ignoring case the values("A", "B", "c")
+
+        fails {
+            expect("aabaabbb") toContain o ignoring case the values("A", "B", "C")
+        }
+    }
+
+    @Test
+    fun valuesIgnoringCaseWithChecker() {
+        expect("ABc") toContain o ignoring case exactly 1 the values("A", "b", "C")
+        expect("AaaBC") toContain o ignoring case atMost 3 the values("A", "B", "c")
+        expect("ABBBcD") toContain o ignoring case atLeast 1 the values("a", "b", "C", "d")
+
+        fails {
+            expect("AAAAAABBBB") toContain o ignoring case atMost 3 the values("a", "b")
+        }
+        fails {
+            expect("AAABBBB") toContain o ignoring case exactly 3 the values("A", "b")
+        }
+        fails {
+            expect("AAAAAABBBB") toContain o ignoring case atLeast 3 the values("a", "b", "C")
+        }
+    }
+
+
+    @Test
+    fun regex() {
+        expect("ABC") toContain o exactly 1 regex "A"
```

Comment:
maybe we should improve those examples (also in fluent) they specify such a simple Regex pattern that it is not different from `value` how about we change this one e.g. to:
"A(BC|C/DC)" ?
Of course this was not part of the issue but maybe you like to improve it in addition?